src/sql/azdata.proposed.d.ts

```diff
@@ -1111,6 +1111,18 @@ declare module 'azdata' {
 			 * The input validation error.
 			 */
 			inputValidationError?: string;
+			/**
+			 * Boolean stating whether it's an edge
```

Comment:
Generally it's not necessary to include the type in a comment, that's already clear from the actual type. Just `Whether it's an edge` is sufficient.

But these seem to be very MSSQL-specific, right? Edge, node and system versioned tables are all MSSQL concepts that I don't think make sense to have in a generic API like this.

Using something like a generic property bag for storing provider-specific properties like these seems like a better option. Then the code below can just check if the values are defined and include them in the events if they are.